### ISSUE TITLE
Wrong Shapefiles over South America

### ISSUE THREAD

**Issue by beneduzi** [matplotlib/basemap#387]
The shapefiles have the wrong state divisions over Brazil.
Using the function mpl_toolkits.basemap.drawstates() in any resolution the state lines are wrong.

**Comment by ocefpaf** [comment]
@beneduzi the data shipped with basemap is dangerously outdated outside USA and Europe. I do not recommend you to use that. You can download updated shapefiles and plot them yourself, like [this examples](https://ocefpaf.github.io/python4oceanographers/blog/2013/08/12/brazil-states-map/), or you can use [cartopy and the natural Earth dataset](https://ocefpaf.github.io/python4oceanographers/blog/2013/09/30/natural_earth/).

**Comment by WeatherGod** [comment]
The data was updated in the v1.1.0 release, which is available through
conda-forge (as opposed to the standard conda channel). Which version are
you using?

**Comment by ocefpaf** [comment]
@WeatherGod v1.1.0 still uses GSHHG 2.3.6, no? As far as I can tell that still does not correctly represent South America. See http://nbviewer.jupyter.org/gist/ocefpaf/93dbaf0fce23ed5c6d4e37fc34a07142

**Comment by WeatherGod** [comment]
Yes, it is using GSHHG 2.3.6. v2.3.7 was released back in June, and its
changenotes is:

"""
Updates the Northern Mariana Islands with CUPS data from NOAA, adds
two missing islands to northern Norway, and adds in the missing
Kosovo-Serbia boundary.
"""

So, it doesn't look like it fixed anything in South America. I would
suggest contacting Paul Wessel to let him know of the problem (feel free to
CC me on the email). He can fix the data and put out a new release, and I
can update it for basemap as well (getting about time for a new release
anyway).

**Comment by ocefpaf** [comment]
> Yes, it is using GSHHG 2.3.6. v2.3.7 was released back in June, and its

Yep. Just tested v2.3.7 and it still has the same issue.

> I would
suggest contacting Paul Wessel to let him know of the problem (feel free to
CC me on the email)

GTM devs are on GitHub :wink:

Pinging @leouieda and @PaulWessel.

**Comment by beneduzi** [comment]
@ocefpaf I use a correct shapefile, but I think a lot of people out of Brazil (and I don't know about the rest of South America), will never notice this kind of error and will use a wrong map. The Political divisions used in this are dated from early 1940's. Thanks @WeatherGod, it would be perfect, I'll contact @PaulWessel.

**Comment by leouieda** [comment]
@beneduzi you have greater chances of success if you include a source for the correct data. Even better if you're willing to try to fix it yourself. The main difficulty is that GSHHG has many different resolutions, all auto generated. So you need to make sure the downscaled versions aren't intersection or crazy in any way. Paul is a busy guy and this will probably not be a huge priority.

**Comment by PaulWessel** [comment]
I see numerous differences between GSHHG and DWC (Digital Chart of the World; also in GMT) for the state divisions of Brazil.  If the DCW are more correct I might be able to replace the GSHHG version with those, unless better data magically arrives.
